tests/test_main.py

```diff
@@ -1,7 +1,5 @@
 import pytest
-import inspect
-
-import app.main
+import inspectimport app.main
```

Comment:
There is a syntax error in the import statement. It should be split into two separate import statements: `import inspect` and `import app.main`.